src/fontbackgroundcolor/fontbackgroundcolorediting.js

```diff
@@ -95,6 +95,11 @@ export default class FontBackgroundColorEditing extends Plugin {
 			columns: 5
 		} );
 
+		editor.config.set(
+			FONT_BACKGROUND_COLOR + '.colors',
+			normalizeColorOptions( editor.config.get( FONT_BACKGROUND_COLOR + '.colors' ) )
+		);
```

Comment:
I don't like the idea of overwriting configuration in a plugin. I think that, instead, the plugin should expose normalized configuration.

Reply:
I just commented on that in the issue. I will quote myself and you can discuss here or there.

> We could also make the normalized config available in a different way (some method or getter) but in my opinion, it doesn't make any sense. It makes it that we have two ways (sources) to get config values. However, honestly, only normalized one is correct and only it should be used. It means that we have a non-obvious way to make an error (using non-normalized config). I also don't see any drawback in re-saving config and don't see any reason to have editor.config be immutable.

Please give me one real reason not to overwrite the config.

Reply:
The problem is that some other plugin might read font configuration but not load the font plugin. It means that his code will not work if the font plugin is not loaded and the configuration is not normalized. If you need to use font plugin API to get the normalized configuration you are forced to load the plugin, it is more straightforward.